src/js/components/sharedComponents/checkbox/AccordionCheckboxSecondary.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * CheckboxOnePrimary.jsx
```

Comment:
need to rename to match the component name

Reply:
renamed